docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md

```diff
@@ -35,8 +35,6 @@ The following list shows most of the technologies that are not available in .NET
 
 In addition to the technologies listed in the official [.NET Core roadmap](https://github.com/dotnet/aspnetcore/wiki/Roadmap), other features might be ported to .NET Core. For a full list, look at the items tagged as [port-to-core](https://github.com/dotnet/corefx/issues?q=is%3Aopen+is%3Aissue+label%3Aport-to-core) on the CoreFX GitHub site. Note that this list does not represent a commitment from Microsoft to bring those components to .NET Core — the items simply capture requests from the community. If you care about any of the components listed above, consider participating in the discussions on GitHub so that your voice can be heard. And if you think something is missing, please [file a new issue in the Runtime repository](https://github.com/dotnet/runtime/issues/new).
 
-Even though .NET Core 3 (at the time of this writing this is in the works) will include support for a lot of existing .NET Framework APIs, these are desktop oriented so, currently, they are of no use in the container world.
-
 ## Using a platform or API that does not support .NET Core
 
 Some Microsoft or third-party platforms do not support .NET Core. For example, some Azure services provide an SDK that is not yet available for consumption on .NET Core. This is temporary, because all Azure services will eventually use .NET Core. For example, the [Azure DocumentDB SDK for .NET Core](https://www.nuget.org/packages/Microsoft.Azure.DocumentDB.Core) was released as a preview on November 16, 2016, but it is now generally available (GA) as a stable version.
```

Comment:
```suggestion
Some Microsoft and third-party platforms don't support .NET Core. For example, some Azure services provide an SDK that isn't yet available for consumption on .NET Core. This is temporary, because all Azure services will eventually use .NET Core. For example, the [Azure DocumentDB SDK for .NET Core](https://www.nuget.org/packages/Microsoft.Azure.DocumentDB.Core) was released as a preview on November 16, 2016, but it is now generally available (GA) as a stable version.
```